docs/tutorial/using-backup-with-syclladb.md

```diff
@@ -0,0 +1,237 @@
+
+<details><summary>Installation command for helm</summary><br>
+
+```bash
+curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
+```
+
+</details>
+
+<details><summary>Installation command for hdf5</summary><br>
+
+```bash
+# yum
+yum install -y hdf5-devel
+
+# apt
+apt-get install libhdf5-serial-dev
+
+# homebrew
+brew install hdf5
+```
+
+</details>
+
+## Prepare the Kubernetes Cluster
+
+1. Prepare Kubernetes cluster
+
+    To complete get started, the kubernetes cluster is required.<br>
+    Vald will run on Cloud Service such as GKE, AWS.
+    In the sense of trying to "Get-Started", [k3d](https://k3d.io/) or [kind](https://kind.sigs.k8s.io/) are easy Kubernetes tools to use.
+
+1. Prepare Scylla DB and kubernetes metrics-server
+
+    Deploy Scylla as a backup database.
+
+    ```bash
+    make k8s/external/scylla/deploy
+    ```
+
+    In this make command, we are deploying a lightweight Cassandra-compatible scylladb using Operator.
+    <details><summary>If you're interested in this make command, take a look here for more detail of make command</summary><br>
+
+    1. Deploy cert-manager for ScyllaDB
+
+    ```bash
+    kubectl apply -f https://github.com/jetstack/cert-manager/releases/latest/download/cert-manager.yaml
+    kubectl wait -n cert-manager --for=condition=ready pod -l app=cert-manager --timeout=60s
+    kubectl wait -n cert-manager --for=condition=ready pod -l app=cainjector --timeout=60s
+    kubectl wait -n cert-manager --for=condition=ready pod -l app=webhook --timeout=60s
+    ```
+
+    1. Deploy ScyllaDB Operator
+
+    ```bash
+    kubectl apply -f https://raw.githubusercontent.com/scylladb/scylla-operator/master/examples/common/operator.yaml
+    kubectl wait -n scylla-operator-system --for=condition=ready pod -l statefulset.kubernetes.io/pod-name=scylla-operator-controller-manager-0 --timeout=600s
+    ```
+
+    1. Deploy ScyllaDB
+
+    ```bash
+    kubectl apply -f k8s/external/scylla/scyllacluster.yaml
+    kubectl wait -n scylla --for=condition=ready pod -l statefulset.kubernetes.io/pod-name=vald-scylla-cluster-dc0-rack0-0 --timeout=600s
+    kubectl -n scylla get pods
+    ```
+
+    1. Configure ScyllaDB
+
+    ```bash
+
+    kubectl apply -f example/manifest/scylla
+    kubectl wait --for=condition=complete job/scylla-init --timeout=60s
+    ```
+
+    </details>
+
+    For documentation on scylladb operator, please refer to [here](http://operator.docs.scylladb.com/master/generic)
+
+1. Apply kubenetes metrics server
+
+    ```bash
+    kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml
+    kubectl wait -n kube-system --for=condition=ready pod -l k8s-app=metrics-server --timeout=600s
+    ```
+
+## Deploy Vald on Kubernetes Cluster
+
+This chapter will show you how to deploy using Helm and run Vald on your kubernetes cluster.<br>
```

Comment:
```suggestion
This chapter will show you how to deploy using Helm and run Vald on your Kubernetes cluster.<br>
```
Should we fix based on the following comment? :thinking:
https://github.com/vdaas/vald/pull/1230#discussion_r633273138